.github/workflows/ci.yml

```diff
@@ -0,0 +1,102 @@
+      WEB_USERNAME_TEST: TravisDashboardUsername
+      PASSWORD_TEST: TravisDashboardPassword
+      SECRET_TEST: TravisCookieSecret
+      DOMAIN_TEST: chatbot-dev.brave.coop
+      PG_USER_TEST: brave
+      PG_DATABASE_TEST: bravebuttons
+      PG_PASSWORD_TEST: travispassword
+      PG_PORT_TEST: 5433
+      PG_HOST_TEST: localhost
+      PG_USER: brave
+      PG_DATABASE: bravebuttons
+      PG_PASSWORD: travispassword
+      PG_PORT: 5433
+      PG_HOST: localhost
+      # Test phone number (from https://www.twilio.com/docs/iam/test-credentials#test-sms-messages-parameters-From)
+      RESPONDER_PHONE_TEST: +15005550006
+      # Test phone number (from https://www.twilio.com/docs/iam/test-credentials#test-sms-messages-parameters-From)
+      STAFF_PHONE_TEST: +15005550006
+      # TWILIO_SID_TEST (from https://www.twilio.com/console/voice/project/test-credentials)
+      TWILIO_SID_TEST: ${{ secrets.TWILIO_SID_TEST }}
+      # TWILIO_TOKEN_TEST (from https://www.twilio.com/console/voice/project/test-credentials)
+      TWILIO_TOKEN_TEST: ${{ secrets.TWILIO_TOKEN_TEST }}
+      PGVER: 12
+      PGPORT: 5433
+      SESSION_RESET_TIMEOUT_TEST: 7200000
+      RADIO_BRIDGE_API_KEY_PRIMARY_TEST: primaryKey
+      RADIO_BRIDGE_API_KEY_SECONDARY_TEST: secondaryKey
+      RAK_API_KEY_PRIMARY_TEST: rakKey1
+      RAK_API_KEY_SECONDARY_TEST: rakKey2
+      PA_API_KEY_PRIMARY_TEST: bravePrimaryKey
+      PA_API_KEY_SECONDARY_TEST: braveSecondaryKey
+
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Set up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ./.nvmrc # node version specified in .nvmrc
+
+      # install dependencies
+      - name: Install dependencies
+        working-directory: ./server
+        run: npm ci
+      
+      # install database
+      - name: Set up PSQL
+        run: |
+          echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
+          wget -qO - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo gpg --dearmor -o /usr/share/keyrings/postgresql-archive-keyring.gpg
+          echo "deb [signed-by=/usr/share/keyrings/postgresql-archive-keyring.gpg] http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
+          sudo apt-get update
+          sudo apt-get install -y postgresql-12 postgresql-client-12
+
+      # ensures that the database is ready before continuing
+      - name: Wait for PostgreSQL to be ready
+        run: |
+          sudo service postgresql restart
+          until pg_isready -h localhost -p 5433 -U brave -d bravebuttons; do
+            echo "Waiting for PostgreSQL to be ready..."
+            sleep 1
+          done
+
+      # setup database
+      - name: Set up database
+        working-directory: ./server
+        run: |
+          ./setup_postgresql_local_dev.sh
```

Comment:
I would recommend copying the commands from this file and having them execute from within the script - it should help you change what's causing the errors you're finding.

Reply:
Note that this file is intended for __local development database setup__, not necessarily Travis. You'll want to setup PG for the GH environment, and then run the setup_postgresql.sh file directly